tests/interfaces/test_qnode_jax.py

```diff
@@ -69,7 +69,7 @@ def circuit(weights):
     # of a numpy array.
     assert "DeviceArray" in grads.__repr__()
     assert grads.shape == (2,)
-    np.testing.assert_allclose(grads, np.array([-0.09784342, -0.19767685]))
+    np.testing.assert_allclose(grads, np.array([-0.09784342, -0.19767685]), atol=tol, rtol=0)

```

Comment:
I think I always had local tests failing here and just ignored them, you too?

Reply:
Yes! The default tolerance for `allclose` is too low, and this test wasn't using the test suite `atol` value 😆